sources/Renderer/Metal/MTEncoderScheduler.mm

```diff
@@ -101,6 +101,7 @@
             renderPassDesc.stencilAttachment.loadAction = MTLLoadActionLoad;
         }
         BindRenderEncoder(renderPassDesc);
+        [renderPassDesc release];
```

Comment:
That one here looks ok to me. I'm sometimes not sure when to use `release` and in Obj-C.